app/models/user.rb

```diff
@@ -18,7 +23,7 @@ def downcase_email
 
   def birthday_must_be_in_last_100_years
     if birthday < 100.years.ago.to_date
-      errors.add(:birthday, t("users.birthday_at_least"))
+      errors.add(:birthday, :birthday_at_least)
```

Comment:
```suggestion
      errors.add :birthday, :birthday_at_least
```
hạn chế  dùng ( nhé. bỏ dc thì bỏ